noise_raster/noise_raster.py

```diff
@@ -298,35 +296,41 @@ def run(self):
                 selectedTableIndex = self.dlg.comboBox.currentIndex()
 
                 # Vectorize energetically added raster including all noise sources
-                out_poly = self.dlg.mQgsFileWidget_outShp.filePath()
+                out_poly = self.dlg.mQgsFileWidget_out.filePath()
                 vectorize(out_final_ras, out_poly, selectedTableIndex)
 
+                # Reproject energetically added raster to EPSG:3035
+                reproject_3035(out_final_ras, out_ras)
+
             else:
 
                 # Write raster in tif format to selected file path
-                out_ras = self.dlg.mQgsFileWidget_outRas.filePath()
+                out_ras = self.dlg.mQgsFileWidget_out.filePath()
 
                 # Merge all input rasters for a single noise source
-                merge_rasters(reprojectlist, out_ras)
+                out_merged_ras = merge_rasters(reprojectlist)
 
                 # Get reclassification table
                 selectedTableIndex = self.dlg.comboBox.currentIndex()
 
                 # Vectorize raster
-                out_poly = self.dlg.mQgsFileWidget_outShp.filePath()
-                vectorize(out_ras, out_poly, selectedTableIndex)
+                out_poly = self.dlg.mQgsFileWidget_out.filePath()
+                vectorize(out_merged_ras, out_poly, selectedTableIndex)
+
+                # Reproject raster to EPSG:3035
+                reproject_3035(out_merged_ras, out_ras)
 
             pass
 
             # Load raster layer in QGIS
-            f_name_ras = os.path.basename(out_ras).split(".")[0]
-            self.iface.addRasterLayer(out_ras, f_name_ras)
+            ras_layer = os.path.join(out_ras, 'final_3035.tif')
```

Comment:
The output file name is replaced by a hard-coded value here. Is this because previously there were two input fields that took paths to file and now there is only one input field that just takes a path to a directory?

Reply:
No, I just want to display the file generated by the ```reproject_3035``` function, whether it is called for a single list of rasters or lists of rasters.